cypress/e2e/tests/migration/applicationinventory/assessment/miscellaneous.test.ts

```diff
@@ -185,6 +199,32 @@ describe(["@tier3"], "Tests related to application assessment and review", () =>
         application2.verifyStatus("review", "Completed");
         application2.validateReviewFields();
 
+        /* Polarion TC 458
```

Comment:
Test steps can be in Polarion unless we want to explain a very difficult flow , Other wise the test script should be explanatory enough.